Examples/DotNetCoreBuild/build-with-casaas.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
```

Comment:
Should this be run as part of our CI to ensure it stays valid?

Reply:
Yes, eventually.  We can do this with or without Kusto logging.  I don't know how expensive (in terms of $$$) it is to pump logs from CI validation builds to Kusto though.